lib/archethic/beacon_chain/summary.ex

```diff
@@ -30,7 +30,8 @@ defmodule Archethic.BeaconChain.Summary do
     node_availabilities: <<>>,
     node_average_availabilities: [],
     end_of_node_synchronizations: [],
-    version: 1
+    network_patches: [],
+    version: 2
```

Comment:
Isn't it better to use a migration task to transform old beacon summaries into the new one with an empty network_patched ?

Reply:
Maybe 🤔